assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -59,11 +61,22 @@ function useUploader(refreshLibrary = true) {
   const uploadFile = (file) => {
     // TODO Add permission check here, see Gutenberg's userCan function.
     if (!fileSizeCheck(file)) {
-      throw new Error('File size error');
+      const SizeError = new Error('File size error');
+      SizeError.name = 'SizeError';
+      SizeError.file = file.name;
+      SizeError.message = `Your file is ${bytesToMB(
```

Comment:
User-facing error messages need to be localized. (It wasn't before because it wasn't user-facing).

Suggestion

```js

SizeError.message = sprintf(
  /* translators: 1: file size. 2: upload limit */
  __('Your file is too big (%1$s). The upload limit is %2$s. Please try again with a smaller file.', 'web-stories'),
  `${bytesToMB(file.size)}MB`,
  `${bytesToMB(maxUpload)}MB`,
));
```

Reply:
Oh, it seems like this message isn't actually used for the snackbar message? 🤔

Reply:
This message is used actually. Let me make it translatable...

Reply:
Fixed